config/initializers/new_framework_defaults_7_0.rb

```diff
@@ -0,0 +1,120 @@
+# Be sure to restart your server when you modify this file.
+#
+# This file eases your Rails 7.0 framework defaults upgrade.
+#
+# Uncomment each configuration one by one to switch to the new default.
+# Once your application is ready to run with all new defaults, you can remove
+# this file and set the `config.load_defaults` to `7.0`.
+#
+# Read the Guide for Upgrading Ruby on Rails for more info on each option.
+# https://guides.rubyonrails.org/upgrading_ruby_on_rails.html
+
+# Temporarily disable this until we clean up the reference data stuff
+Rails.configuration.active_record.belongs_to_required_by_default = false
```

Comment:
Is this actually default?
If not can we move to its lonesome?

Reply:
How bad is turning this on? It should probably live in `applicaiton.rb` rather than keeping this file alive

Reply:
I think i'd prefer to keep it here since this file is supposed to help with easing the upgrade. This is usually on by default but had to turn it off as it requires refactoring of https://github.com/DFE-Digital/teacher-training-api/blob/main/app/services/subjects/creator_service.rb#L2 which creates subjects without subject areas. Maybe it should be combined with this file https://github.com/DFE-Digital/teacher-training-api/blob/main/app/services/subjects/subject_area_creator_service.rb to achieve the same thing. But keeping it here means we can just remove it once we've fixed it.

Ideally this should be on, i wouldn't want to put it anywhere else which could mean we've decided that it shouldn't be true.

Reply:
Sounds good as long as we have a card in the backlog for it